weblate/vcs/git.py

```diff
@@ -816,35 +804,50 @@ def _get_version(cls):
     def get_username():
         return settings.GITLAB_USERNAME
 
-    def create_pull_request(self, origin_branch, fork_remote, fork_branch):
-        """Create merge (a.k.a pull) request.
+    def api_url(self):
+        repo_name_index = self.component.repo.rfind(
+            "/", 0, self.component.repo.rfind("/")
+        )
+        api_url = "{}/api/v4/projects/{}".format(
+            self.component.repo[:repo_name_index],
+            urllib.parse.quote(
+                self.component.repo[(repo_name_index + 1) :].replace(".git", ""),
+                safe="",
+            ),
+        )
+        return api_url
 
-        Used to merge branch in forked repository into branch of remote
-        repository.
+    def create_fork(self):
+        fork_url = "{}/fork".format(self.api_url())
+        r = requests.post(
+            fork_url,
+            headers={"Authorization": "Bearer {}".format(settings.GITLAB_TOKEN)},
+            data={},
+        )
```

Comment:
Same remark as with GitHub - what happens in case fork exists? Either the code should contain comments describing the behavior or handle this case.

Reply:
Seems gitlab doesn't handle the fork case. It returns a message saying `"path": ["has already been taken"]`. So in GitLab side, will have to handle this case.